src/core/BBoxOffset.tsx

```diff
@@ -0,0 +1,42 @@
+import * as React from 'react'
+import { Box3, Vector3, Object3D } from 'three'
```

Comment:
Lets make this `import * as THREE from 'three'` as all other components use that. Just consistancy

Reply:
Sure thing